### ISSUE TITLE
Increase admin boundary error threshold

### ISSUE THREAD

**Issue by DenisCarriere** [osmlab/wikidata-osm#18]
Seems like a lot of the "issues/errors" that are reported are admin boundaries, which don't have the same location precision as a town or a city.

Would it be possible to change the classification by increasing the error threshold of features that contain `boundary=administrative`.

- <50 (Good)
- 50 - 75 (Decent)
- 75 - 100 (Bad)
- 100+ (Very bad)

Any thoughts?

**Comment by planemad** [comment]
👍 also seeing this for geographical features like rivers and lakes.

Not sure what a scalable solution is, maybe have some preset filters to only show features based on a tag?

**Comment by DenisCarriere** [comment]
Yes Geographical features have the same issue (ex: National parks).

I think having a preset filter would work, also be able to change the error threshold based on min & max slider or manual inputs.
